test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -403,5 +403,35 @@ public void ConcatWith()
             Assert.Equal(NumberType.Float, types[2].ItemType);
             Assert.Equal(NumberType.Float, types[3].ItemType);
         }
+
+        [Fact]
+        public void Tokenize()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
+            var dataPath = GetDataPath("wikipedia-detox-250-line-data.tsv");
```

Comment:
>h("wikipedia-detox-250-line-data.tsv" [](start = 37, length = 37)

should use the datasets from the TestDatasets.cs

Reply:
Sorry, I merged before I saw your comments. Will do in one of the next PRs.

---
In reply to: [218631455](https://github.com/dotnet/machinelearning/pull/931#discussion_r218631455) [](ancestors = 218631455)